main_app/templates/main_app/register.html

```diff
@@ -47,83 +47,133 @@
-                    <h1 class="display-4" style="text-align: center; color: black; font-family: 'Bangers', cursive; ">
-                        Register</h1>
-                    <link href="https://fonts.googleapis.com/css2?family=Overpass:wght@300&display=swap"
-                        rel="stylesheet">
-                    <p class="lead" style="font-family: 'Overpass', sans-serif; text-align: center;"><b>Create Your
-                            Account</b></p>
-
-                    <!-- <img src="{% static 'Images/signup-image.jpg' %}" alt="" srcset="" style=""> -->
-                    <form method="POST" class="form-style" style="background-color: white; margin-top: 5%;   ">
-                        <img src="{% static 'Images/signup-image.jpg' %}" alt="Sign Up"
-                            style="margin-left: 32%; height: 35%; width: 35%;"><br><br>
-                        {% csrf_token %}
-                        {{form.as_p}}
-                        <br>
-                        <div style=" margin-top: 3rem; display: flex; align-items: center;
-                    justify-content: center;">
-                            <button class="btn btn-success" style=" border-radius: 25px;"
-                                type="submit"><b>Register</b></button>
-                        </div>
-                        <br>
-                        <br>
-                        <div style="display: flex; align-items: center;
-                    justify-content: center;">
-                            <strong>Already on RESCUE? </strong><a href="/login" style="color: rgb(30,142,62);"><b
-                                    class="sign-in"><u>Sign
-                                        In</u></b></a>
-                        </div>
-                    </form>
+                
+                <h1 class="register-page-title">WELCOME TO RESCUE</h1>
+                <p class="register-page-text">Create Your Account</p>
+                        
+                <img class="container register-page-img" src="{% static 'Images/signup-image.svg' %}" alt="Sign Up">
+                
+                <form method="POST" class="form-style">
+                    {% csrf_token %}
+                    {{form.as_p}}
+                    
+                    <br>
 
-                </div>
-                <br>
-                <br>
+                    <div class="register-sign-in">
+                        <input type="submit" value="Register">
+                        <h3 class="register-sign-in-text">Already on Rescue? <a href="/login">Sign In</a></h3>
+                    </div>
+                </form>
+                
             </div>
         </div>
 
         <style>
-            @media screen and (max-device-width: 768px) {
+            @import url('https://fonts.googleapis.com/css2?family=Bebas+Neue&display=swap');
+            /* Register Page styling */
+            .register-page{
+                margin-top: 12em;
```

Comment:
Can you please make `margin-top:6em;` only? I thought it would look better on 12em but 6em is perfect.

Reply:
margin-top 10em is perfect else the title goes under the navbar